scripting/get5/mapveto.sp

```diff
@@ -14,6 +14,8 @@ public void CreateVeto() {
   g_VetoCaptains[MatchTeam_Team1] = GetTeamCaptain(MatchTeam_Team1);
   g_VetoCaptains[MatchTeam_Team2] = GetTeamCaptain(MatchTeam_Team2);
   ResetReadyStatus();
+  if (g_FreezePlayersVetoCvar.BoolValue) {
+    Pause(); }
```

Comment:
@TandelK And fix format here. Bracket down a line. Same in VetoFinished below.

Reply:
> @TandelK Can you rename the cvar accordingly, since it's no longer a freeze?

Its still technical Freezing right or should i rename it to
`g_PausePlayersVetoCvar.BoolValue`

Reply:
@nickdnk I have resolved both the above mentioned comments.